src/Components/GEOSldas_GridComp/GEOSlandassim_GridComp/clsm_ensupd_read_obs.F90

```diff
@@ -7645,6 +8156,283 @@ subroutine scale_obs_tskin_zscore( N_catd, tile_coord,   &
     
   end subroutine scale_obs_tskin_zscore
 
+ 
+  ! *****************************************************************
+  
+  subroutine scale_obs_sfmc_zscore( N_catd, tile_coord,         &
+       date_time, this_obs_param, tmp_lon, tmp_lat, tmp_time,   &
+       tmp_obs, tmp_std_obs )
+    
+    ! scale sfmc or sfds obs to model climatology via standard-normal-deviate (zscore)
+    ! scaling using seasonally varying (pentad) stats
+    ! Grid information is read from a NetCDF file   
+    ! 
+    ! Scaling parameters are read from a NetCDF file that contains the following:
+    !  variables:
+    !  int version ;
+    !  double ll_lon ;
+    !          ll_lon:standard_name = "longitude of lower left corner" ;
+    !  double ll_lat ;
+    !          ll_lat:standard_name = "latitude of lower left corner" ;
+    !  double d_lon ;
+    !          d_lon:standard_name = "longitude grid spacing" ;
+    !  double d_lat ;
+    !          d_lat:standard_name = "latitude grid spacing" ;
+    !  int pentad(pentad) ;
+    !          pentad:standard_name = "pentad" ;
+    !  double start_time(pentad) ;
+    !          start_time:standard_name = "start time" ;
+    !  double end_time(pentad) ;
+    !          end_time:standard_name = "end time" ;
+    !  double lon(lon) ;
+    !          lon:standard_name = "longitude" ;
+    !  double lat(lat) ;
+    !          lat:standard_name = "latitude" ;
```

Comment:
This is where the meaning of 'lon' and 'lat' as the **lower** **boundaries** of the grid cells is lost.  See my comment re. `write_netcdf_latlon_grid.m`